osv/repos.py

```diff
@@ -115,23 +117,14 @@ class RepoInaccessibleError(Exception):
 def clone(git_url, checkout_dir, git_callbacks=None, blobless=False):
   """Perform a clone."""
   try:
-    # Use 'git' CLI here as it's much faster than libgit2's clone.
-    env = _set_git_callback_env(git_callbacks)
-    cmd = ['git', 'clone']
-    if blobless:
-      cmd.append('--filter=blob:none')
-    cmd.extend([_git_mirror(git_url), checkout_dir])
-    subprocess.run(cmd, env=env, capture_output=True, check=True)
+    os.makedirs(checkout_dir, exist_ok=True)
+    cmd = ['curl', f"{GITTER_HOST}/getgit?url="+ _git_mirror(git_url), "-o", checkout_dir+".zst", "-s"]
```

Comment:
few things:
1. nit: single quotes for python strings
2. nit: instead of concatenating:`f'{GITTER_HOST}/getgit?url={_git_mirror(git_url)}'`
3. I doubt `_git_mirror` / `git_url` is safe to use as a query param without encoding?